commands.js

```diff
@@ -1,34 +1,45 @@
 'use strict';
 const fs = require('fs');
 const chalk = require('chalk');
-const pathFile = 'input.json';
+const path = require('path');
+const os = require('os');
 
-function createIfNotExists(path) {
+const pathFile = getSavePath();
+
+function getSavePath() {
+  const currentDir = os.homedir();
+  const pth = 'SHORTC_PATH' in process.env ?
+    process.env['SHORTC_PATH'] : currentDir;
```

Comment:
```suggestion
  const pth = process.env['SHORTC_PATH'] || currentDir;
```

Reply:
What about the cases, when `process.env['SHORTC_PATH'] == ""`? Or in this case it doesn't matter?

Reply:
If it's empty string I'd take currentDir too )